ChangeLog.md

```diff
@@ -23,7 +136,7 @@ and some new interesting features. Here is the brief summary of changes:
   - Wingman now gives a warning if it ran out of gas during "attempt to fill hole".
   - Metaprogramming for Wingman has been improved with symbolic-name support and the `pointwise` combinator.
   - An option to enable/disable Wingman's proof state styling is added.
-  - Hole fit suggestions are now disabled for performance reasons when using Wingman. 
```

Comment:
This one is good though (one whitespace does not have any special effect in markdown AFAIK)